includes/options/views/class-amp-analytics-options-serializer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+class Analytics_Options_Serializer {
+
+	public static function save() {
+
+		$option_name = 'analytics';
+
+		if ( !( empty( $_POST['vendor-type'] ) ||
+		     empty( $_POST['config'] ) ) ) {
+
+			if ( empty( $_POST['id-value'] ) ) {
+				$_POST['id-value'] = md5( $_POST['config'] );
+			}
+
+			$new_analytics_options = array(
+				$_POST['id-value'],
+				$_POST['vendor-type'],
```

Comment:
How should we do this? Checking a list of allowed types? Wonder if we should let the user ensure that the type entered is correct, since right now they are doing it manually. WDYT?